whisperv6/peer.go

```diff
@@ -135,6 +140,10 @@ func (peer *Peer) handshake() error {
 		return fmt.Errorf("peer [%x] is useless: two light client communication restricted", peer.ID())
 	}
 
+	egressCfg := ratelimiter.Config{}
```

Comment:
Let's add some code what we are doing here. Like: 
```
// Decode received egress configuration from the peer. In case it's not defined,
// sending data to that peer won't be limited.
```